src/Table/Table.tsx

```diff
@@ -23,6 +24,11 @@ interface Props<RowShape> {
    * Definition of how each column will be rendered
    */
   columns: readonly {
+    /**
+     * Apply text-align all the cells in this column
+     */
+    align?: CSS.TextAlignProperty;
```

Comment:
I think it'd be useful to allow specifying this for the column header as well  
I think most often we'd want the alignment to match, but can't discount the possibility of needing them to differ

Reply:
I agree, the text alignment has to do with the entire column. I can't think of a case where I'd want different alignment for the header and the content.

Reply:
Oh, and, left align should be the default. I don't think it would be otherwise, but just wanted to state that.